Makefile.pre.in

```diff
@@ -646,6 +646,20 @@ all:		@DEF_MAKE_ALL_RULE@
 # all.
 .PHONY: all
 
+# Provide quick help for common Makefile targets.
+.PHONY: help
+help:
+	@echo "Please use \`make <target>' where <target> is one of"
+	@echo "  clean      to remove build files"
+	@echo "  distclean  'clean' + remove other generated files (patch, exe, etc)"
+	@echo "  clinic     run Argument Clinic over source files"
+	@echo "  install    install built files"
+	@echo "  recheck    rerun configure with last cmdline options"
+	@echo "  regen-all  regenerate a number of generated source files"
+	@echo "  reindent   reindent .py files in Lib directory"
+	@echo "  tags       build a tags file (useful for Emacs and other editors)"
+	@echo "  test       run the test suite"
```

Comment:
Rather than sorting by name, maybe sort by logical order. Maybe something like:

* make regen-all
* make
* make test
* ...
* make clean
* make distclean
* make tags
* make reindent
* ...

Reply:
A couple people have asked for a grouped list. I'll add something which seems logical to me in the next edit.